src/screen-cast.c

```diff
@@ -100,6 +114,146 @@ GType screen_cast_session_get_type (void);
 
 G_DEFINE_TYPE (ScreenCastSession, screen_cast_session, session_get_type ())
 
+static void
+set_persistent_permissions (const char *app_id,
+                            const char *restore_token,
+                            GVariant   *restore_data)
```

Comment:
sneaked in some fancy variable name alignment I see ;P

Reply:
A mistake